src/ring_theory/principal_ideal_domain.lean

```diff
@@ -342,6 +342,44 @@ begin
       (of_irreducible_mul h).resolve_left nu) }
 end
 
+theorem is_coprime_of_irreducible_dvd {x y : R}
+  (z : ¬ (x = 0 ∧ y = 0))
+  (H : ∀ z : R, irreducible z → z ∣ x → ¬ z ∣ y) :
```

Comment:
You have a hypothesis called `z`, and then reuse it as name for an element of `R` on the next line. I suggest renaming the hypothesis.

Reply:
Done. (This name dates back to #527, fwiw.)